### ISSUE TITLE
Race - Volus Thinktank

### ISSUE THREAD

**Issue by queryluke** [queryluke/masseffect-5e#244]
Pulling from #236 

Via bpmcpherson:
>Additionally, is there a way to make the Volus' Savvy Dealings ability have more of a rules backing? It's hard to determine how that will work.

Via Nmmoriarty:
>Volus' Savvy Dealings somewhat locks anyone playing them into being the team's designated shopper. It is a tricky game balance thing, especially since this system differs from DnD 5th substantially in this arena: there are actually things to spend your hard earned gol-- I mean, credits on. On top of that, given the games infrastructure, refilling your consumables is more an issue of access than an issue of cost. My original thought was their Savvy Dealings ability would drop the difficulty of acquiring rarer items, but this can have some unintended consequences.
>* Scenario 1 is that the campaign is a "scum and villainy" style where the players are not members of a military outfit, so it becomes a requirement to have a Volus in the party if you want to reliably have access to military grade gear (though grenades, in their current state, start to lose their potency after level 6, but that's another issue).
>* Scenario 2 is you are a military outfit (as is traditional for ME games), and access to military class gear is paltry. Volus characters effectively won't use this trait.
>
>I'm not sure I like how either shakes out. The other alternatives is that Volus start with more credits (but to balance that we'd have to look at what the expected wealth curve per level is), or a solid discount on buying things (which makes them designated shopper). If we go discount, I think we need to add a limiter that they can get an item 5% to 15% cheaper (again, not sure on the wealth curve), but only once per week/month/shopping trip/break between missions so players can't abuse it. We could make this more powerful if we also gave the Volus a penalty trait because of their ammonia based physiology, a thing were their suit needs to be upkept/repaired if critically hit or they start to suffocate.

Via Duneday:
>Volus are an interesting point, but ill start with the concrete facts. While the gravity from their planet is roughly 1.5x earth it is explicitly mentioned that "they are not physically adept compared to most species, volus mostly make their influence felt through trade and commerce." Making the justification for a +1 str a little flimsy. A Cha bonus would be more lore friendly if not mechanically stunted (Cha Void) or maybe even an Int bonus would be more suiting. Savvy dealings could be great it could be awful, either way it does need a solid mechanic. I have to disagree with Nmmoriarty here though, in purely meta sense he would be correct and Volus could become a designated shopper, but that completely ignores character which should always be a driving force in games. I believe a Volus would be a little resentful most of the times and probably derive no small amount of enjoyment in getting better deals than those around him (a bit of a napoleon complex). Ultimately I'm of the belief that the solid discount is the best option even in meta situations of "designated shopper" as a racial it may accelerate party growth slightly, but it is still at the behest of a GM as to what becomes available when and how much money a party has access to in addition to what equipment they would have access to. That said the point of the suit drawback is a valid one, it could function effectively as the Quarian one requiring an action to seal after an unprotected crit.

Via Nmmoriarty:
>For the Volus, a solid discount is fine, but I still feel like it breaks game balance too much, especially if the discount is high. It needs a limiter of some variety to keep players from being 5% to 20% richer simply because there is a Volus around -- and without a limiter, every party will want a Volus. Volus will be essentially left in support roles, but that's not super bad, but a little niche from a design perspective.

Via Duneday:
>In terms of Volus, its still a GM issue at that point Stores have stock and its not likely theyll keep super expensive things on hand, such things usually need to be ordered and pass through red tape, there are sooo many options for a GM to reign in spending its kind of a nonissue. As characters they seem like theyd have issues fitting into a combat squad. From my perspective being a short fat money grubber doesn't appeal as a player.

**Comment by bpmcpherson** [comment]
Is there, perhaps, a way to grant a bonus to persuasion when attempting to use it to barter? It's not a straight-up "have a discount", but does reflect the mercantile skill of the race and the pride they have for it.

**Comment by SrShdw** [comment]
I'd say just change their ability bonuses to Wisdom and Intelligence. They aren't particularly Charismatic from most interactions you have for them, they're just smart and able to read people well. Advantage or proficiency with Persuasion and Insight would be a good replacement for Savvy Dealings IMO. I'd suggest Proficiency over advantage as it lines up more with other racial bonuses from 5e. 

I'm fine with them having Fury of the Small, and I would normally suggest giving them something like the Halfling's Lucky ability, but since most races seem to get feats, giving them the actual Feat seems fine.

**Comment by SrShdw** [comment]
Just proficiency is would be fine, you don't need to overcompensate.

**Comment by bpmcpherson** [comment]
Seems like we've got a pretty sound solution, then.

**Comment by Nmmoriarty** [comment]
Looks good to me.

**Comment by queryluke** [comment]
Putting a button on this then, but leaving discussion open for a few days for final comments.

### Proposed Change:
Ability Scores improvements: +2 Wis, +1 Int
Savvy Dealings: Proficiency with the Persuasion skill

**Comment by Duneday** [comment]
Im happy with this. The attribute change is in line with what I had envisioned in my original post and happy to see savvy dealings simplified and concrete.
